atlas/src/shared/components/TextField/TextField.tsx

```diff
@@ -1,25 +1,15 @@
-import React, { useState, useRef, useEffect } from 'react'
-import { useCSS, TextFieldStyleProps } from './TextField.style'
-import { IconProp } from '@fortawesome/fontawesome-svg-core'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useEffect, useRef, useState } from 'react'
```

Comment:
We never use this component anywhere, and if we decided to use it, it would need to reworked anyway. I suggest we just get rid of it for now

Reply:
Removed 👍